docs/api-client/README.md

```diff
@@ -1404,3 +1404,96 @@ Example
   ]
 }
 ```
+
+## Private Instances
+
+### getGithubToken
+
+Retrieves Github personal access token stored in your environment variables. Used for automated private instance creation to push yaml files (associated with private instance you're creating) to `openlaw-infra` repo.
+
+::: warning Authorization
+This resource can only be accessed by a logged in user with an `Admin` role as further explained in the [toAdminUser method](#toadminuser).
+:::
+
+```
+GET /getGithubToken
+```
+
+**Parameters**
+None.
```

Comment:
```suggestion

None
```
added a line break here so it renders on a new line and removed the period. again just consistency with rest of doc.